style.css

```diff
@@ -334,3 +350,146 @@ textarea {
   margin-top: 20px;
   margin-bottom: 18px;
 }
+
+@media screen and (min-width: 1024px) {
+  .fa-bars::before {
+    display: none;
+  }
+
+  .nav-color {
+    padding: 10px 12px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-top: 25px;
+  }
+
+  .nav-link {
+    color: #344563;
+  }
+
+  .logo {
+    margin-top: 25px;
+    padding-left: 100px;
+  }
+
+  .nav-list {
+    display: flex;
+    padding-right: 100px;
+  }
+
+  main {
+    background-image: url(./images/header-bg-desktop.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: top right;
+    padding: 0 250px 0 250px;
+  }
+
+  .profile {
+    font-size: 48px;
+    width: 400px;
+    line-height: 60px;
+    padding-top: 200px;
+    color: #091e42;
+  }
+
+  .dev {
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    width: 100%;
+  }
+
+  .pix {
+    display: none;
+  }
+
+  .snap {
+    display: unset;
+  }
+
+  .bon {
+    margin-top: 50px;
+  }
+
+  .card-img {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .card-content {
+    padding-left: 20px;
+    padding-right: 20px;
+    display: flex;
+    flex-direction: column;
```

Comment:
- [ ] Please try to follow the given design in all the aspects (typography, font size and weight, colors, and spacings such as margin and padding).
> Please look at the images down below and make the corresponding changes.
**Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/121583802-0457c000-ca31-11eb-8df5-9ed67babf9d2.png)
**[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121583909-29e4c980-ca31-11eb-82fb-0fbc9cb87678.png)